patches/chromium/printing.patch

```diff
@@ -264,7 +264,22 @@ index e4cacbf9a82da8dca062cc33df885a0575e74830..69f6a0947679a474d12dcc179e400ebd
    if (!job_settings.FindIntKey(kSettingPrinterType)) {
      UpdatePrintSettingsReply(std::move(callback),
                               CreateEmptyPrintPagesParamsPtr(), false);
-@@ -722,13 +742,17 @@ void PrintViewManagerBase::PrintingFailed(int32_t cookie) {
+     return;
+   }
+ 
++#if 0
```

Comment:
We are disabling pdf rasterization https://github.com/electron/electron/blob/cbeae20438be8baf03ce0e32a73563a80aeccb37/shell/browser/api/electron_api_web_contents.cc#L2548 , the change follows suit with it.